src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const page = document.querySelector('body');
+
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve(compleat);
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(error);
+  }, 3000);
+});
+
+promise1
+  .then((result) => {
+    result();
+  });
+
+promise2
+  .catch((result) => {
+    result();
+  });
+
+const compleat = () => {
+  page.insertAdjacentHTML('beforeend',
+    `<div class = 'message'>Promise was resolved!</div>`);
+};
+
+const error = () => {
+  page.insertAdjacentHTML('beforeend',
+    `<div class = 'message error-message'>Promise was rejected!</div>`);
+};
```

Comment:
would be better to create `showMesssage `function wich will take message and show it

Reply:
then you can resolve/reject promises with messages and run this function in `then`. It will unify showing message logic